tests/functional/spec/features/calculated_summary/calculated_summary.spec.js

```diff
@@ -15,3 +15,18 @@ describe("Feature: Calculated Summary", () => {
     CalculatedSummaryTestCase.testCrossSectionDependencies("test_new_calculated_summary_cross_section_dependencies.json");
   });
 });
+
+describe("Feature: Calculated Summary with negative values", () => {
+  describe("Given I enter a negative value in the first section", () => {
+    CalculatedSummaryTestCase.testNegative("test_calculated_summary.json", -1, 2, 3, 0, 4, "-£1.00");
```

Comment:
I don't think you need to include verifying `-£1.00` in the test as its not part of the summary, just a single answer, I think the 4 answers and expected total suffice  (would change the total to be a formatted string though, see below)

Reply:
I thought that the "-£1.00" would be used to check if the summary page contains it, meaning that the summary page "allows" negatives and it displays correctly. I attached an image to where I thought it would check.


<img width="697" alt="Screenshot 2023-09-28 at 15 47 27" src="https://github.com/ONSdigital/eq-questionnaire-runner/assets/144226112/640e42d9-75fe-4ce9-9326-139016304101">

Reply:
Ahhh I see, yeah I think thats a good idea, what about changing it to a list, `expectedAnswerValues` so that with your other test cases where the negative value is not the first answer but another its verified too. Theres a helper method `assertSummaryValues` which you could use for this

Reply:
Yep, that worked and looks good!

Reply:
Happy this is resolved.